geth/node/manager.go

```diff
@@ -34,14 +37,17 @@ var (
 // NodeManager manages Status node (which abstracts contained geth node)
 type NodeManager struct {
 	sync.RWMutex
-	config         *params.NodeConfig // Status node configuration
-	node           *node.Node         // reference to Geth P2P stack/node
-	nodeStarted    chan struct{}      // channel to wait for start up notifications
-	nodeStopped    chan struct{}      // channel to wait for termination notifications
-	whisperService *whisper.Whisper   // reference to Whisper service
-	lesService     *les.LightEthereum // reference to LES service
-	rpcClient      *rpc.Client        // reference to RPC client
-	rpcServer      *rpc.Server        // reference to RPC server
+	config            *params.NodeConfig // Status node configuration
+	node              *node.Node         // reference to Geth P2P stack/node
+	nodeStarted       chan struct{}      // channel to wait for start up notifications
```

Comment:
Maybe we could move these channels at the and of other fields? It would be just better grouping I think.